src/ducks/Auth.duck.js

```diff
@@ -48,6 +53,10 @@ const initialState = {
   // signup
   signupError: null,
   signupInProgress: false,
+
+  // confirm (create use with idp)
+  confirmErro: null,
```

Comment:
can I confirm, Is this a typo?

Reply:
@feorgie yes that's a typo, it should be `confirmError`. Thanks for letting us know!

Reply:
no problem!